Source/Tx.Windows/PerfCounters/PerfCounterRealTimeProbe.cs

```diff
@@ -55,7 +55,10 @@ public void OnTimer(object state)
 
         public override void Dispose()
         {
-            _timer.Dispose();
+            if (_timer != null)
+            {
+                _timer.Dispose();
+            }
 
             base.Dispose();
```

Comment:
@Expecho, you are absolutely right, in this case your fix covers the issue exactly as you described. The rest seems to be out if scope and to be handled separately.